assets/build/js/app/postBabel/common/dcf-dialog.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
This appears to be a temporary build file. I would avoid committing artifacts like this to git. Perhaps add remove these and then add a line to the `.gitignore` file?

Reply:
Build file will be excluded from distribution, pre-transpiled and pre-minified module files are now included inside of dist